rotkehlchen/chain/ethereum/tokens.py

```diff
@@ -72,19 +71,15 @@ def detect_tokens_for_address(
     ) -> Dict[EthereumToken, FVal]:
         balances: Dict[EthereumToken, FVal] = defaultdict(FVal)
         if self.ethereum.connected_to_any_web3():
-            call_order = []
-            if NodeName.OWN in self.ethereum.web3_mapping:
-                call_order = [NodeName.OWN]
+            # For all the chunks we use the same order of the nodes
+            call_order = self.ethereum.default_call_order()
```

Comment:
Why change the logic here? We always gave priority to own node, if connected

Reply:
Yes, the default also prioritizes the own node. Is first returned and then the others https://github.com/yabirgb/rotki/blob/public-nodes/rotkehlchen/chain/ethereum/manager.py#L294

Reply:
okay this is not obvious at all from the code. The code there needs to become more readable. I commented there.